.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: khcaxtwPaTT1iUbcptGAPag8uqO2lWPFf
```

Comment:
do it through env var and add to docker-compose

Reply:
✔️